src/interface/report/Results/PlayerInfoConduits.tsx

```diff
@@ -0,0 +1,30 @@
+import { ConduitLink } from 'interface/index';
+import { Conduit } from 'parser/core/Events';
+
+interface PlayerInfoConduitsProps {
+  conduits: Conduit[] | undefined;
+}
+
+const PlayerInfoConduits = ({ conduits }: PlayerInfoConduitsProps) => (
+  <div className="player-details-talents">
+    <h3>Conduits </h3>
```

Comment:
I'd remove the extra whitespace. Wouldnt make much difference but looks cleaner :)

Reply:
Agreed for sure, it was just there in the original code so I left it as is. Will clean up some of the legacy code.